### ISSUE TITLE
Set up email subscription

### ISSUE THREAD

**Issue by xaqrox** [radcampaign/winrock2023#22]
In FR:

> List of email addresses that receive email notifications when a story
> is published to the site. On the new site, replace the plugin with
> WordPress's subscriber role. We only want to shoot emails to
> subscribers when we publish news, stories, blogs, and press releases.

I'm not sure this is the best implementation -- does Wordpress have a
way to manage email subscriptions? It looks like wordpress*.com* does,
but built into WP core?

**Comment by thunderdw** [comment]
I think I had thought that it did when that got in there. Maybe a good solution would be to migrate their story subscribers to a MailChimp list with an RSS-to-email feed
https://mailchimp.com/features/rss-to-email/

**Comment by xaqrox** [comment]
Hmmm, maybe we do need to follow up about the requirements for this. flagging for @kristinsj

**Comment by kristinsj** [comment]
@thunderdw @xaqrox was there an issue with the existing functionality that we are solving for? Happy to pitch the alternate Rss-to-email idea, but just want to be sure we're not trying to fix something that's not broken?

**Comment by thunderdw** [comment]
@kristinsj yeah good point, I think it probably does make sense to just do the same plugin as before https://wordpress.org/plugins/email-subscribers/
